app/services/notifications/mentioned_in_comment_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Notifications
+  class MentionedInCommentService
+    def self.model
+      Notifications::MentionedInComment
+    end
+
+    def self.notify(mentionable, recipient_user_ids)
```

Comment:
- Metrics/AbcSize: Assignment Branch Condition size for notify is too high. [<6, 28, 6> 29.26/20]
- Metrics/MethodLength: Method has too many lines. [25/20]